### ISSUE TITLE
Design: Terms and conditions page

### ISSUE THREAD

**Issue by darshi-shah** [saurabhd/hk_realestate#75]
This page is in 2 column layout.

**Comment by heliogabal** [comment]
Yes, I think this should be in 2 column layout to make it more compressed and readable, much like the existing site. As this doesn't need any further design, I think it is safe to also move this along, as there would be just a paragraph to put the terms text into two columns. So again, if you agree, move this to approved.

**Comment by darshi-shah** [comment]
No design needed for this.
